### PR TITLE
setup mocha configuration

### PR DESCRIPTION
**Pull request by Lilainita**

### CODE REVIEW

**Review by kliver98 — CHANGES_REQUESTED** [review]
Steps are missing:

- Add also package.json file generated when run command `npm init` (Step 8)
- Step 9 depends on step 8
- Add to the README file the text __This is a Workshop about Api Testing in JavaScript__

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @holgiosalos @danielgalvis98 @kliver98 @AlejaGonzalezV @NicolasB2 @manuelq12 @Valeryibarra @veronicatofino
```
Comment (manuelq12) [inline .github/CODEOWNERS:1]: EOF

File: test/HelloWord.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const { assert } = require('chai');
+
+describe('Array', () => {
+    describe('#indexOf()', () => {
+        it('should return -1 when the value is not present', function () {
+            assert.equal(-1, [1, 2, 3].indexOf(4));
+        });
+    });
+});
```
Comment (manuelq12) [inline test/HelloWord.test.js:9]: EOF

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+*.cab
+*.msi
+*.msix
+*.msm
+*.msp
+
+# Windows shortcuts
+*.lnk
+
+# End of https://www.toptal.com/developers/gitignore/api/windows,visualstudiocode,node
```
Comment (manuelq12) [inline .gitignore:196]: EOF

**Review by kliver98 — COMMENTED** [review]
Looks good to me, wait for Manuel to approve his comments

**Comment by manuelq12** [comment]
@Lilainita Please take care of the EOF to approve

File: .gitignore (hunk 3 above)
Comment (Lilainita) [reply .gitignore:196]: Hello Manuel, 
What do you mean with EOF (End of File)?
Do I need to do something else?

**Comment by Lilainita** [comment]
> @Lilainita Please take care of the EOF to approve

I don't understand what you mean with that, can you explain?

**Comment by manuelq12** [comment]
> > @Lilainita Please take care of the EOF to approve
> 
> I don't understand what you mean with that, can you explain?

The following is an EOF "End of file" it means that the file you are working on is missing for **One** line-break, it is a good practice because is easier for the compiler to read the files

Screen Shot 2022-06-20 at 5 31 41 PM

**Review by manuelq12 — APPROVED** [review]

**Review by kliver98 — APPROVED** [review]

**Pull request merged**